src/App/Expression.tab.c

```diff
@@ -1,3 +1,5 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
Confirming: even though Bison is GPL3, the special exception clause allows us to distribute the generated parser under any license we choose.